lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -131,6 +131,10 @@ sub test_network_interface {
             } else {
                 script_run("ssh root\@$guest systemctl restart network", 300);
             }
+            # Check if it's a Tumbleweed test and restart Network Manager if so
```

Comment:
Quick question, why do restart Network Manager service here? Would you mind adding some comments or info here? Thanks

Reply:
No IP found for enX1 in opensuse-tumbleweed-64-pv-xen at /var/lib/openqa/pool/5/os-autoinst-distri-opensuse/lib/virt_autotest/virtual_network_utils.pm line 151.

Got the above error
maybe this restart is not required, and have not tested it yet, and wild guess that a restart of NetworkManager would solve it.

I'll remove it if not required.

Reply:
Indeed.  Honestly, If there are no special needs or workarounds, `restart service` should be reduced as much as possible during our test execution. Meanwhile, the current code has some problems left over from history, and they need to be further optimized and solved.